changelogs/fragments/170-add_ignore_txt.yml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - Add ingore-2.12.txt.
```

Comment:
This Looks to be a typo.

Reply:
@rohitthakur2590 fixed thanks. Aren't trivial fragments ignored anyway?

Reply:
I'm not sure about this collection, so interesting.